addons/account/views/account_payment_term_views.xml

```diff
@@ -77,10 +77,10 @@
                                     Example:
                                     <field name="example_amount" class="oe_inline"/>
                                     on
-                                    <field name="example_date" class="fw-bold oe_inline"/>
+                                    <field name="example_date" class="fw-bold border-bottom oe_inline" style="width: 20% !important;"/>
                                 </div>
-                                <div colspan="2" class="py-1 bg-view">
-                                    <field name="note" placeholder="e.g. Payment terms: 30 days after invoice date"/>
+                                <div colspan="2" class="py-1 bg-secondary">
+                                    <field name="note" placeholder="e.g. Payment terms: 30 days after invoice date" class="border-bottom border-gray note_align hide-draggable ps-2 pe-2"/>
```

Comment:
maybe instead of putting that many classes we can put just one that apply all the styling